ihmc-avatar-interfaces/src/main/java/us/ihmc/avatar/networkProcessor/kinematicsToolboxModule/KinematicsToolboxPrivilegedConfigurationParameters.java

```diff
@@ -0,0 +1,106 @@
+package us.ihmc.avatar.networkProcessor.kinematicsToolboxModule;
+
+import us.ihmc.commonWalkingControlModules.configurations.JointPrivilegedConfigurationParameters;
+
+public class KinematicsToolboxPrivilegedConfigurationParameters extends JointPrivilegedConfigurationParameters
+{
+   public KinematicsToolboxPrivilegedConfigurationParameters()
+   {
+   }
+
+   /**
+    * To compute the nullspace of the high-level control objectives, a matrix inversion has to be
+    * calculated. To improve stability, a damped-least square pseudo-inverse is used, which in turn
+    * uses this alpha parameter.
+    * <p>
+    * It should preferably be close to zero.
+    * </p>
+    * 
+    * @return the parameter used for computing the damped-least squares pseudo-inverse.
+    */
+   public double getNullspaceProjectionAlpha()
+   {
+      return 0.005;
```

Comment:
Copied-pasted code and doc and not the best. Here since it appears that the default values have not been changed, I'd suggest leaving this class empty, that'll simply redirect to the super class and won't change anything